beacon-light-client/plonky2/crates/circuits/src/types.rs

```diff
@@ -0,0 +1,46 @@
+use crate::{
+    validators_commitment_mapper::first_level::ValidatorsCommitmentMapperFirstLevel,
+    withdrawal_credentials_balance_aggregator::first_level::WithdrawalCredentialsBalanceAggregatorFirstLevel,
+};
+use circuit::CircuitOutput;
+use serde::{Deserialize, Serialize};
+
+// TODO: the corresponding typescript types don't have public inputs field
```

Comment:
This needs to be done before merging as it will create problems if we have multiple workers and some pick up an empty proof added by the typescript